src/components/date-picker/DatePicker.stories.tsx

```diff
@@ -259,6 +277,64 @@ export const withRange = () => {
     );
 };
 
+export const withRangeAndKeyboardInput = () => {
```

Comment:
Is a new story needed or can we update the `withRange` story above to support the knob with a default value of `true`? It seems like we should try to avoid having too many permutations of the base story.

Reply:
I was having difficulty with adding two default knobs to `withRange`, but maybe it was just a naming conflict, will try again!

Reply:
So it turns out knobs & state addons aren't compatible, and it also turns out knobs have been deprecated in favor of controls which is something to look into at a later date. Changing a knob value calls forceReRender which messes with state renders. Didn't notice when testing since I initially tested by just adding isKeyboardInputAllowed to all the stories before I remembered that knobs are a thing.

Instead I've removed the knobs and added two stories with isKeyboardInputAllowed defaulted to true.